### PR TITLE
Add some "manual" System.Console tests

### PR DESCRIPTION
**Pull request by stephentoub**
Today our test suite for System.Console is significantly lacking, due to our current inability to handle reading, verification of output to the terminal, etc.  Until we are better at such things, this PR adds a manual test suite, that instructs the user to do something and to then verify specific visual results and behaviors.

I'm submitting this as its own PR, but added it as part of validating a bug fix I'm submitting separately.

cc: @ianhays

### CODE REVIEW

File: src/System.Console/tests/ManualTests/ManualTests.cs [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System
+{
+    public class ConsoleManualTests
+    {
+        public static bool ManualTestsEnabled => !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("MANUAL_TESTS"));
```
Comment (ianhays) [inline src/System.Console/tests/ManualTests/ManualTests.cs]: If these only run when an environment variable is set anyways, why not add them to the existing System.Console test proj? Requiring the env variable when the tests are their own project seems like a hassle.

**Comment by ianhays** [comment]
This is related to https://github.com/dotnet/corefx/issues/9777

**Comment by stephentoub** [comment]
> This is related to #9777 - maybe add a test for that scenario?

Sure, good idea.  I'll do that separately, since it's a pure addition to the suite.

Thanks for reviewing.

File: src/System.Console/tests/ManualTests/ManualTests.cs (hunk 1 above)
Comment (ianhays) [inline src/System.Console/tests/ManualTests/ManualTests.cs]: Ah, sounds reasonable. Thanks for the explanation.

**Pull request merged**